### DISCUSSION TITLE
2021/11/15/tg-qq-gocq/

### DISCUSSION THREAD

**Discussion by giscus** [xzsk2/xzsk2.github.io#3] [Announcements]
# 2021/11/15/tg-qq-gocq/

使用 Bot 在 Telegram 及 QQ 间转发消息，基本可以做到去 QQ 化，仅在 Telegram 上与 QQ 好友/群组互动

https://sakari.top/2021/11/15/tg-qq-gocq/

**Comment by PBnicad** [comment]


**Reply by simpread** [reply]
github的说明里不是有写么：在本地直接下载使用 go-cqhttp 配置运行，登陆成功后将生成的 device.json 上传到远程服务器对应位置重新登陆

**Comment by simpread** [comment]
(๑ơ ₃ ơ)ﻌﻌﻌ♥ (感谢大佬，还是docker的好用，省去好多麻烦哦！╭(′▽`)╭(′▽`)╯

**Comment by sbbbsbbb** [comment]
请问是不是没办法适配ARM的机器

**Reply by sakarie9** [reply]
试着发了个 arm64 的

把 `docker-compose.yml` 中的 `image: xzsk2/gocqhttp-docker:latest` 这行替换成 `image: xzsk2/gocqhttp-docker:arm64` 试试

**Reply by sbbbsbbb** [reply]
感谢更新。
需要把 image: xzsk2/efb-docker:latest 也编译成arm的 就可以用了

**Reply by sakarie9** [reply]
`xzsk2/efb-docker:latest` 和 `xzsk2/gocqhttp-docker:latest ` 都加了 arm64 支持了，方便的话可以测试一下吗

**Reply by sbbbsbbb** [reply]
已测试，可以使用。无任何警告信息。

**Comment by EasonHuang1212** [comment]
感谢大佬，但是我还是有个问题，这个自动更新终端运行之后返回/bin/sh: docker: not found，不知道该咋用

**Reply by sakarie9** [reply]
docker没装好吧，重装一遍试试

**Comment by linep47** [comment]
你好 请问目前不能收到消息，可以发消息，出现这个的情况是刚开始可以收到，没多久（时间有长有短）就会收不到。

**Reply by sbbbsbbb** [reply]
目前还是无法接收到信息
